packages/components/src/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -42,18 +42,31 @@ function Breadcrumbs(props) {
 	 * @returns {*} Breadcrumb item rendering depending of its position
 	 */
 	function renderBreadcrumbItem(item, index) {
-		if (maxItemsReached && index < ellipsisIndex) {
-			return null;
-		}
 		const { text, title, onClick } = item;
 		const isActive = index === nbItems - 1;
 		const id = `${props.id}-item-${index}`;
 		const separator = index < props.items.length - 1 && (
-			<li className="separator" key={`${index}-separator`}>
+			<li
+				className={classNames('tc-breadcrumb-separator', 'separator')}
+				key={`${index}-separator`}
+				aria-hidden="true"
+			>
 				<Icon name="talend-chevron-left" transform="rotate-180" />
 			</li>
 		);
 
+		function getItemContent() {
+			return onClick ? (
+				<Button id={id} bsStyle="link" role="link" title={title} onClick={wrappedOnClick}>
+					{text}
+				</Button>
+			) : (
+				<span id={id} title={title}>
+					{text}
+				</span>
+			);
+		}
+
```

Comment:
I prefer to define a variable here. you have the if onclick just after and a lint error on wrappedOnClicl

```javascript
                let wrappedOnClick;
                let itemContent = <span id={id} title={title}>{text}</span>;
 		if (onClick) {
 			wrappedOnClick = event => onClick(event, item);
                         itemContent = <Button...
 		}
```